lib/index.js

```diff
@@ -2,6 +2,8 @@ var fs = require('fs'),
 	path = require('path'),
 	colors = require('colors'),
 	common = require('./common'),
+	// Windows PowerShell translates magenta to the same blue as the background. So use white, instead.
+	inputAndMaskColor = process.title.indexOf('PowerShell') >= 0 ? 'white' : 'magenta',
```

Comment:
Yeah, I agree -- the title is a bit crazy, but I'm not sure how else to detect it. I looked around for a bit and didn't find other differences. Maybe there is a better way? Anyway, yeah, adding the process.platform check could help.